New Kusto version (2019-09-07)

liatbezalel opened the pull request:
Generated and added tests by the new Kusto 2019-09-07 version.
This related to the following PRs:

https://github.com/Azure/azure-rest-api-specs/pull/7219
https://github.com/Azure/azure-rest-api-specs/pull/7300

Commit:
Genereated the new client

Commit:
Added tests for the new version

Commit:
Recorded the tests

Commit:
Updated version

isra-fel commented on sdk/kusto/Microsoft.Azure.Management.Kusto/src/Microsoft.Azure.Management.Kusto.csproj:
```diff
@@ -5,7 +5,7 @@
+    <Version>3.0.0</Version>
     <AssemblyName>Microsoft.Azure.Management.Kusto</AssemblyName>
     <PackageTags>management;kusto</PackageTags>
     <PackageReleaseNotes>
```
PackageReleaseNotes needs to be updated

isra-fel reviewed (changes requested):
@liatbezalel please link your swagger spec review PR(s) here & update release notes, thanks

isra-fel commented:
@liatbezalel ping on this

Commit:
Added tests

Commit:
Added package release notes

liatbezalel commented:
> @liatbezalel please link your swagger spec review PR(s) here & update release notes, thanks

I updated the release notes and added the links to the relevant PRs in the description.
Thanks
